packages/docs/content/blog/release-8.0.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Release 8.0
+date: 2023-09-18
+intro: Normalizing CSS, standardizing spacing & colors, a new Dropdown, and more
```

Comment:
we don't talk about normalizing css do we?

Reply:
Ah ha, no